printer_mods/Zen3D/V0_Skirt/README.md

```diff
@@ -0,0 +1,46 @@
+# Voron0 dual color skirt set (SSR, PowerSwitch, Nameplate)
+
+To match the skirt with the [Midpanel with Hex Pattern](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Zen3D/V0_MidPanel_HexPattern)  by sterminatore's idea
```

Comment:
Change this to a relative link

    [Midpanel with Hex Pattern](../V0_MidPanel_HexPattern)